lib/api/route/messages.dart

```diff
@@ -1,64 +1,16 @@
 import 'package:json_annotation/json_annotation.dart';
 
 import '../core.dart';
-import '../exception.dart';
 import '../model/model.dart';
 import '../model/narrow.dart';
 
 part 'messages.g.dart';
 
-/// Convenience function to get a single message from any server.
-///
-/// This encapsulates a server-feature check.
-///
-/// Gives null if the server reports that the message doesn't exist.
-// TODO(server-5) Simplify this away; just use getMessage.
-Future<Message?> getMessageCompat(ApiConnection connection, {
```

Comment:
nit in commit message:

    See also: 631f4d68c58a84dff66a6a9a80322584f0d60152

Instead say:

    See also commit 631f4d68c58a84dff66a6a9a80322584f0d60152.

That way it's clear this hex blob refers to a commit (implicitly one in this same repo).